importexport/musicxml/exportxml.cpp

```diff
@@ -3239,7 +3239,7 @@ static void writePitch(XmlWriter& xml, const Note* const note, const bool useDru
       // `alter2` represents a microtone or manually-adjusted note tuning.
       // In MusicXML, These two values are merged in the same "alter" tag.
       if (alter || alter2)
-            xml.tag("alter", (double)alter + alter2);
+            xml.tag("alter", double(alter + alter2));
```

Comment:
What happens if you drop that cast? If that doesn't result in a compiler warning, just drop it.
The default result of any arithmetic involving a `double` and an `int` is to convert the result into a `double` anyhow.
See also https://docs.microsoft.com/en-us/cpp/cpp/standard-conversions?view=msvc-160#arithmetic-conversions

Reply:
Thanks, dropping the cast also seems to pass the tests.